src/react/deckgl.js

```diff
@@ -138,23 +138,49 @@ export default class DeckGL extends React.Component {
       events: {
         click: this._onClick,
         mousemove: this._onMouseMove,
-        dragstart: this._onDragEvent,
-        dragmove: this._onDragEvent,
-        dragend: this._onDragEvent,
-        dragcancel: this._onDragCancel
+        panstart: this._onDragEvent,
+        panmove: this._onDragEvent,
+        panend: this._onDragEvent,
+        pancancel: this._onDragEvent
       }
     });
   }
 
+  // Get mouse position {x, y} relative to the container
+  _getPos(event) {
+    const {srcEvent, center, rootElement} = event;
+    let x;
+    let y;
+
+    if (center) {
```

Comment:
Another vote for cracking Hammer open to do what we want it to (per our conv today)...it's annoying that our application layer needs to know about pointer vs. mouse events. Ideally we could just query the event for some standardized `x` and `y`.